lib/nerves_time/rtc/abracon/id.ex

```diff
@@ -4,19 +4,22 @@ defmodule NervesTime.RTC.Abracon.ID do
   @doc """
   Return a list of commands for reading the ID registers
   """
+
+  use Bitwise
+
   def reads() do
     # Register 0x28-0x2E
     [{:write_read, <<0x28>>, 7}]
   end
 
   @spec decode(<<_::16>>) :: {:ok, map()} | {:error, any()}
   def decode(
-        <<0x18, 0x05, 0x13, lot::integer-8, lot9::integer-1, part_id::big-integer-15,
+        <<id0, id1, id2, lot::integer-8, lot9::integer-1, part_id::big-integer-15,
           lot8::integer-1, wafer::integer-5, _::integer-2>>
       ) do
     {:ok,
      %{
-       id: :ab_rtcmc_32768khz_ibo5_s3,
+       id: <<id0 &&& 0x08, id1, id2>>,
```

Comment:
⚠️ (recommended)

Based on the prior source code's naming scheme, it looks like this ID would be something like `:ab_rtcmc_32768khz_08x5` for the AB08X5 part.